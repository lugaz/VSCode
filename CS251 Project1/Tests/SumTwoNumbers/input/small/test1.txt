111799950381093675898603409919629518598261
725477506963135028
