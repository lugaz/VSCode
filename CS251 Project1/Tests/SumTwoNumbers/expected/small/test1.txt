111799950381093675898604135397136481733289
